src/biomappings/templates/home.html

```diff
@@ -59,10 +59,14 @@ <h2 class="text-center">Biomappings Curation Interface</h2>
                     </tbody>
                 </table>
                 <div class="panel-footer">
+                    <a href="{{ url_for('home', limit=limit, offset=0) }}">First</a> |
                     {% if 0 <= offset - limit %}
                         <a href="{{ url_for('home', limit=limit, offset=offset - limit) }}">Previous {{ limit }}</a> |
                     {% endif %}
-                    <a href="{{ url_for('home', limit=limit, offset=offset + limit) }}">Next {{ limit }}</a>
+                    {% if offset <= controller.total_predictions - limit %}
+                        <a href="{{ url_for('home', limit=limit, offset=offset + limit) }}">Next {{ limit }}</a> |
+                    {% endif %}
+                    <a href="{{ url_for('home', limit=limit, offset=controller.total_predictions - limit) }}">Last</a>
```

Comment:
The `Next {limit}` and `Last` don't go away when you're on the last page, but I wouldn't spend much time worrying about that